src/FlatFeeCalculator.sol

```diff
@@ -24,6 +24,12 @@ contract FlatFeeCalculator is IFeeCalculator, Ownable {
 
     uint256 public feeBasisPoints = 300;
 
+    /// @dev The scale between the decimals of the fee token and the underlying token.
+    /// For example, if the fee token has 18 decimals and the underlying token has 18 decimals, the
+    /// decimals scale is 1e0.
+    /// If the fee token has 18 decimals and the underlying token has 0 decimals, the decimals scale is 1e18
+    uint256 public feeToUnderlyingDecimalsScale = 1e0;
```

Comment:
shouldn't this be 1e18?

Reply:
Default is for case where TCO2 and POOL has same amount of decimals, hence the diff / scale is 1 (1e0).

Reply:
but this calculator is only used with REDD  and all its values are specific for REDD right (including the feeBasisPoints)? So it would make sense to initialize it to a value that make sense for this use case as well
, I believe